nav.css

```diff
@@ -104,18 +104,49 @@
 	padding-left: var(--ncn-pad-h);
 	padding-right: var(--ncn-pad-h);
 }
+.netlify-logo-encapsulated {
+	display: none;
+}
+:where(.netlify-logo-full) {
+	--_netlify-logo-lines-color: var(--netlify-logo-lines-color, #05bdba);
+	--_netlify-logo-text-color: var(--netlify-logo-text-color, #014847);
+}
+
+.netlify-logo-full[data-theme="dark"] {
+	--_netlify-logo-lines-color: #32E6E2;
+	--_netlify-logo-text-color: #ffffff;
+}
+:where(.netlify-logo-encapsulated) {
+	--_netlify-logo-lines-color: var(--netlify-logo-lines-color, #32E6E2);
+	--_netlify-logo-text-color: var(--netlify-logo-text-color, #ffffff);
+	--_netlify-logo-fill: var(--netlify-logo-fill, #014847);
+}
+.netlify-logo-encapsulated[data-theme="dark"] {
+	--_netlify-logo-lines-color: #05BDBA;
+	--_netlify-logo-text-color: #014847;
+	--_netlify-logo-fill: #fff;
+}
+.netlify-logo-encapsulated[data-theme="monochrome-dark"] {
+	--_netlify-logo-lines-color: #0C2A2A;
+	--_netlify-logo-text-color: #0C2A2A;
+	--_netlify-logo-fill: #014847;
+  }
+
 @media (max-width: 26.1875em) { /* 419px */
 	.ncn-home-logo {
-		min-width: 4.6875em; /* 75px /16 */
+		--ncn-pad-h: .5em;
 	}
-	.ncn-home-logo-text {
+	.netlify-logo-full {
 		display: none;
 	}
+	.netlify-logo-encapsulated {
+		display: block;
+	}
 }
 @media (min-width: 26.25em) { /* 420px /16 */
 	.ncn-home-logo {
 		width: auto;
-		height: 1.5625em; /* 25px /16 */
+		height: 2.375em; /* 38px /16 */
```

Comment:
Curious how changing this height affects the block size of the navigation and how that might impact the way it looks on the answers.netlify.com site with the sticky container

Reply:
This height only applies to the Logo. It doesn't actually affect the height of the nav bar. Both are `68px` in height.